app/models/ability.rb

```diff
@@ -0,0 +1,15 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new # guest user (not logged in)
+
+    if user.admin?
+      can :manage, :all
```

Comment:
- [ ] You are calling the method `admin?` from the user model but this method is not present there. Please make sure to create this method in the user model.
  - You can consider using the below code:
  
```ruby
 def admin?
     role == 'admin' ? true : false
   end
```